configs/config-test.json

```diff
@@ -76,6 +76,12 @@
       "verticleInstances": 1,
       "required":["postgresOptions", "commonOptions", "keycloakOptions"],
       "poolSize": "25"
+    },
+    {
+      "id": "iudx.aaa.server.apd.ApdVerticle",
+      "verticleInstances": 1,
+      "required":["postgresOptions", "commonOptions"],
+      "poolSize": "25"
```

Comment:
@karun-singh this may need to be added to the unit test config `config-test.json`. Integration tests have not been updated, so no changes needed there.

Reply:
Okay, I've updated the test-config with the same. 
You can try building the pipeline again.